backend/server.py

```diff
@@ -0,0 +1,63 @@
+import db_interface
+from flask import Flask, jsonify, request, Response, json
```

Comment:
Since this will be a REST server and not a web server, we should use flask_restful rather than vanilla flask.

Reply:
should discuss this in a future meeting when we move on to phase 1 - professor